packages/web3-providers/src/types/RSS3.ts

```diff
@@ -189,13 +189,11 @@ export namespace RSS3BaseAPI {
         author: string[]
         created_at?: string
         target_url?: string
-        media: [
-            {
-                /** URL */
-                address: string
-                mime_type: 'image/png' | string
-            },
-        ]
+        media: Array<{
+            /** URL */
+            address: string
+            mime_type: 'image/png' | string
```

Comment:
Make the editor happy.

```suggestion
            mime_type: 'image/png' | Omit<string, 'image/png'>
```

Reply:
Got such error:
<img width="1218" alt="image" src="https://user-images.githubusercontent.com/1141198/226240149-b1804f83-fa16-4682-8bdb-5bf8b092b0a2.png">

Reply:
`'image/png' | string` is equal to `string`

Reply:
> Got such error:
> 
> <img width="1218" alt="image" src="https://user-images.githubusercontent.com/1141198/226240149-b1804f83-fa16-4682-8bdb-5bf8b092b0a2.png">
> 
> 

Omit will help auto completion. Maybe cast it to string?

Reply:
> `'image/png' | string` is equal to `string`

I know, just want to document it in code.

Reply:
Would like to list as many types as possible.